### PR TITLE
fix(mongodb): improving mongodb connectionId parsing + verbose logging

### PR DESCRIPTION
**Pull request by galbash**

### CODE REVIEW

File: src/events/mongodb.js [hunk 1]
```diff
@@ -24,21 +24,49 @@ function stringifyData(obj) {
...
 function onStartHook(event) {
     try {
         const startTime = Date.now();
+        utils.debugLog('Epsagon - MongoDB - handling event', event);
 
-        let { host, port } = event.connectionId;
-        if (!host) {
-            [host, port] = event.connectionId.split(':');
-        }
+        const { host, port } = getConnectionDetails(event ? event.connectionId : null);
```
Comment (ranrib) [inline src/events/mongodb.js:66]: If there's no event is there any point to continue at all? Maybe check if event.connectionId?

File: src/events/mongodb.js [hunk 2]
```diff
@@ -24,21 +24,49 @@ function stringifyData(obj) {
...
+ */
+function getConnectionDetails(connId) {
+    let host;
+    let port;
+    utils.debugLog('Epsagon - MongoDB - inspecting connectionId', connId);
+    if (connId) {
+        if (typeof connId === 'string') {
+            const parts = connId.split(':');
+            if (parts.length && parts[0][0] === '/') {
+                host = 'localhost';
```
Comment (ranrib) [inline src/events/mongodb.js:40]: Can you add a quick explaination what is this scenario? (What's the original strong looks like)

File: src/events/mongodb.js (hunk 1 above)
Comment (galbash) [reply src/events/mongodb.js:66]: not really possible, just a safeguard

File: src/events/mongodb.js (hunk 2 above)
Comment (galbash) [reply src/events/mongodb.js:40]: in case it is a unix domain socket (i.e a file on the FS)

**Review by ranrib — APPROVED** [review]

**Review by tvaintrob — APPROVED** [review]

**Pull request merged**